### PR TITLE
Added docs and antora support

### PR DESCRIPTION
**Pull request by patilsuraj767**
Rendered version of docs can be found at - http://dhcp131-80.gsslab.pnq2.redhat.com/yuptoo/dev/index.html

### CODE REVIEW

File: docs/modules/ROOT/pages/ephemeral_environment.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Login to the cluster by grabbing a login command from https://oauth-openshift.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/oauth/token/request[token request page] for the ephemeral cluster
+----
+oc login --token=sha256~xxxxxx --server=https://api.c-rh-c-eph.8p0c.p1.openshiftapps.com:6443
+----
+
+Run bonfire command to reserve the ephemeral namespace and to deploy yuptoo in that namespace
+----
+bonfire deploy yuptoo
+----
+Above command will return the name of namespace which has been reserved, make a note of it.
```
Comment (kgaikwad) [inline docs/modules/ROOT/pages/ephemeral_environment.adoc:16]:
```suggestion
Note down the name of reserved namespace from the output of above bonfire command. 
```

File: docs/modules/ROOT/pages/yuptoo_yupana.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+== Yuptoo (Yupana Upload Processor II)
+
+Yuptoo is a rewrite of yupana service. Below are a key differences and improvement made in yuptoo.
```
Comment (kgaikwad) [inline docs/modules/ROOT/pages/yuptoo_yupana.adoc:3]:
```suggestion
Yuptoo is a rewrite of Yupana service. Below are the key differences and improvements made in yuptoo. 
```

File: docs/modules/ROOT/pages/tar_archive_details.adoc [hunk 3]
```diff
@@ -0,0 +1,142 @@
+= Yuptoo specific tar archive
+
+Yuptoo requires a specially formatted tar.gz file. Files that do not conform to the required format will be marked as invalid and no processing will occur. The tar.gz file must contain a metadata JSON file and one or more report slice JSON files. The file that contains metadata information is named metadata.json, while the files containing host data are named with their uniquely generated UUID4 report_slice_id followed by the .json extension.
```
[the remark this answers is not in the document]
Comment (patilsuraj767) [reply docs/modules/ROOT/pages/tar_archive_details.adoc]: No, it also supports bzip2 and xz. Done the required modification in doc.

**Review by kgaikwad — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thank you @patilsuraj767 and @r14chandra  for review.

**Pull request merged**